sdk/storage/azure-storage-file-datalake/tests/test_file_system_async.py

```diff
@@ -1,20 +1,16 @@
-# coding: utf-8
-
 # -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
 import asyncio
+
 import pytest
```

Comment:
Move `pytest` down on top of `azure.core` (not built-in) and remove new line to join `asyncio` and `unittest`.

Reply:
Doh. This one always gets me. Will fix 😄